diff -rupN --no-dereference gdal-3.4.3-fedora/m4/lib-link.m4 gdal-3.4.3-fedora-new/m4/lib-link.m4
--- gdal-3.4.3-fedora/m4/lib-link.m4	2022-04-22 11:03:37.000000000 +0200
+++ gdal-3.4.3-fedora-new/m4/lib-link.m4	2022-05-04 16:04:12.870079834 +0200
@@ -108,8 +108,6 @@ dnl   acl_hardcode_direct,
 dnl   acl_hardcode_minus_L.
 AC_DEFUN([AC_LIB_RPATH],
 [
-  dnl Tell automake >= 1.10 to complain if config.rpath is missing.
-  m4_ifdef([AC_REQUIRE_AUX_FILE], [AC_REQUIRE_AUX_FILE([config.rpath])])
   AC_REQUIRE([AC_PROG_CC])                dnl we use $CC, $GCC, $LDFLAGS
   AC_REQUIRE([AC_LIB_PROG_LD])            dnl we use $LD, $with_gnu_ld
   AC_REQUIRE([AC_CANONICAL_HOST])         dnl we use $host
