diff -rupN --no-dereference gdal-3.4.3-fedora/frmts/iso8211/GNUmakefile gdal-3.4.3-fedora-new/frmts/iso8211/GNUmakefile
--- gdal-3.4.3-fedora/frmts/iso8211/GNUmakefile	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/frmts/iso8211/GNUmakefile	2022-05-04 16:04:11.552196892 +0200
@@ -23,8 +23,7 @@ dist-clean:	clean
 	rm -rf $(DISTDIR)
 
 $(ISOLIB):	$(OBJ:.o=.$(OBJ_EXT))
-	$(AR) r $(ISOLIB) $?
-	$(RANLIB) $(ISOLIB)
+	$(SHELL) $(top_builddir)/libtool --mode=link gcc -static -o $(ISOLIB) $?
 
 8211createfromxml$(EXE):	8211createfromxml.$(OBJ_EXT)
 	$(LD) $(LDFLAGS) 8211createfromxml.$(OBJ_EXT) $(CONFIG_LIBS) -o 8211createfromxml$(EXE)
