diff -rupN --no-dereference gdal-3.4.3-fedora/swig/java/add_javadoc.c gdal-3.4.3-fedora-new/swig/java/add_javadoc.c
--- gdal-3.4.3-fedora/swig/java/add_javadoc.c	2022-04-22 11:02:06.000000000 +0200
+++ gdal-3.4.3-fedora-new/swig/java/add_javadoc.c	2022-05-04 16:04:11.092237798 +0200
@@ -142,21 +142,21 @@ int main(int argc, char* argv[])
     JavaDocInstance* instances = (JavaDocInstance*)calloc(sizeof(JavaDocInstance), 3000);
     int nInstances = 0;
     char szLine[512];
-    char szClass[256];
+    char szClass[512];
     char javadoc[16384];
     szClass[0] = 0;
-    while(fgets(szLine, 255, fSrc))
+    while(fgets(szLine, 511, fSrc))
     {
         if (strstr(szLine, "/**") == NULL) continue;
 begin:
         strcpy(javadoc, szLine);
-        while(fgets(szLine, 255, fSrc))
+        while(fgets(szLine, 511, fSrc))
         {
             strcat(javadoc, szLine);
             if (strstr(szLine, "*/"))
                 break;
         }
-        while(fgets(szLine, 255, fSrc))
+        while(fgets(szLine, 511, fSrc))
         {
             if (szLine[0] == 10)
                 continue;
@@ -209,10 +209,10 @@ begin:
             continue;
         }
         szClass[0] = 0;
-        char szPackage[256];
+        char szPackage[512];
         szPackage[0] = 0;
 
-        while(fgets(szLine, 255, fSrc))
+        while(fgets(szLine, 511, fSrc))
         {
             char szMethodName[1024];
             char* szOriLine = strdup(szLine);
@@ -255,7 +255,7 @@ begin:
                     strcpy(szMethodName, szLine);
                     do
                     {
-                        ignore_ret(fgets(szLine, 255, fSrc));
+                        ignore_ret(fgets(szLine, 511, fSrc));
                         strcpy(szMethodName + strlen(szMethodName) - 1, szLine);
                     } while (!strchr(szMethodName,')'));
                     strcpy(szLine, szMethodName);
@@ -282,7 +282,7 @@ begin:
                             {
                                 do
                                 {
-                                    ignore_ret(fgets(szLine, 255, fSrc));
+                                    ignore_ret(fgets(szLine, 511, fSrc));
                                 } while (!strchr(szLine,'}'));
                             }
                             break;
@@ -310,7 +310,7 @@ begin:
                                         nBrackets --;
                                     }
                                 }
-                                ignore_ret(fgets(szLine, 255, fSrc));
+                                ignore_ret(fgets(szLine, 511, fSrc));
                             } while(bFoundOpen == FALSE || nBrackets > 0);
                         }
                         else
diff -rupN --no-dereference gdal-3.4.3-fedora/swig/java/javadoc.java gdal-3.4.3-fedora-new/swig/java/javadoc.java
--- gdal-3.4.3-fedora/swig/java/javadoc.java	2022-04-22 11:02:07.000000000 +0200
+++ gdal-3.4.3-fedora-new/swig/java/javadoc.java	2022-05-04 16:04:11.094237621 +0200
@@ -11982,12 +11982,12 @@ public interface osrConstants:public fin
 public interface osrConstants:public final static int PROJ_ERR_COORD_TRANSFM
 
 /**
- * Invalid input coordinate. e.g a latitude > 90°.
+ * Invalid input coordinate. e.g a latitude > 90 deg.
  */
 public interface osrConstants:public final static int PROJ_ERR_COORD_TRANSFM_INVALID_COORD
 
 /**
- * Coordinate is outside of the projection domain. e.g approximate mercator with \|longitude - lon_0\| > 90°,
+ * Coordinate is outside of the projection domain. e.g approximate mercator with \|longitude - lon_0\| > 90 deg,
  * or iterative convergence method failed.
  */
 public interface osrConstants:public final static int PROJ_ERR_COORD_TRANSFM_OUTSIDE_PROJECTION_DOMAIN
diff -rupN --no-dereference gdal-3.4.3-fedora/swig/java/java.opt gdal-3.4.3-fedora-new/swig/java/java.opt
--- gdal-3.4.3-fedora/swig/java/java.opt	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/swig/java/java.opt	2022-05-04 16:04:11.094237621 +0200
@@ -7,6 +7,6 @@ JAVADOC=$(JAVA_HOME)/bin/javadoc
 JAVAC=$(JAVA_HOME)/bin/javac
 JAVA=$(JAVA_HOME)/bin/java
 JAR=$(JAVA_HOME)/bin/jar
-JAVA_INCLUDE=-I$(JAVA_HOME)/include -I$(JAVA_HOME)/include/linux
+JAVA_INCLUDE=-I/usr/lib/jvm/java/include/ -I/usr/lib/jvm/java/include/linux
 
 
