diff -rupN --no-dereference gdal-3.4.3-fedora/configure.ac gdal-3.4.3-fedora-new/configure.ac
--- gdal-3.4.3-fedora/configure.ac	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/configure.ac	2022-05-04 16:04:12.651099272 +0200
@@ -73,7 +73,7 @@ dnl warning #2304: non-explicit construc
 dnl warning #3280: declaration hides member
 dnl remark #11074: Inlining inhibited by limit max-size
 dnl remark #11076: To get full report use -qopt-report=4 -qopt-report-phase ipo
-AX_CHECK_COMPILE_FLAG([-diag-disable 188,1684,2259,2304,3280,11074,11076],[C_WFLAGS="$C_WFLAGS -diag-disable 188,1684,2259,2304,3280,11074,11076" CXX_WFLAGS="$CXX_WFLAGS -diag-disable 188,1684,2259,2304,3280,11074,11076"],,[$ERROR_ON_UNKNOWN_OPTIONS])
+#AX_CHECK_COMPILE_FLAG([-diag-disable 188,1684,2259,2304,3280,11074,11076],[C_WFLAGS="$C_WFLAGS -diag-disable 188,1684,2259,2304,3280,11074,11076" CXX_WFLAGS="$CXX_WFLAGS -diag-disable 188,1684,2259,2304,3280,11074,11076"],,[$ERROR_ON_UNKNOWN_OPTIONS])
 
 AX_CHECK_COMPILE_FLAG([-Wextra],[C_WFLAGS="$C_WFLAGS -Wextra" CXX_WFLAGS="$CXX_WFLAGS -Wextra"],,[$ERROR_ON_UNKNOWN_OPTIONS])
 AX_CHECK_COMPILE_FLAG([-Winit-self],[C_WFLAGS="$C_WFLAGS -Winit-self" CXX_WFLAGS="$CXX_WFLAGS -Winit-self"],,[$ERROR_ON_UNKNOWN_OPTIONS])
