diff -rupN --no-dereference gdal-3.4.3-fedora/apps/GNUmakefile gdal-3.4.3-fedora-new/apps/GNUmakefile
--- gdal-3.4.3-fedora/apps/GNUmakefile	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/apps/GNUmakefile	2022-05-04 16:04:11.984158476 +0200
@@ -232,6 +232,7 @@ gdal-config-inst:	gdal-config.in ../GDAL
 
 install: default
 	for f in $(BIN_LIST) ; do $(INSTALL) $$f $(DESTDIR)$(INST_BIN) ; done
+	for f in $(BIN_LIST) ; do $(INSTALL) .libs/$$f $(DESTDIR)$(INST_BIN) ; done
 	$(INSTALL_DATA) gdal_utils.h $(DESTDIR)$(INST_INCLUDE)
 	$(INSTALL) gdal-config-inst $(DESTDIR)$(INST_BIN)/gdal-config
 
