diff -rupN --no-dereference gdal-3.4.3-fedora/frmts/dods/GNUmakefile gdal-3.4.3-fedora-new/frmts/dods/GNUmakefile
--- gdal-3.4.3-fedora/frmts/dods/GNUmakefile	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/frmts/dods/GNUmakefile	2022-05-04 16:04:11.324217168 +0200
@@ -4,7 +4,7 @@ include ../../GDALmake.opt
 
 OBJ	=	dodsdataset2.o
 
-CPPFLAGS	:=	 $(CPPFLAGS) $(DODS_INC) $(LIBXML2_INC)
+CPPFLAGS	:=	 $(CPPFLAGS) $(DODS_INC) $(LIBXML2_INC) -I/usr/include/tirpc
 
 default:	$(OBJ:.o=.$(OBJ_EXT))
 
diff -rupN --no-dereference gdal-3.4.3-fedora/ogr/ogrsf_frmts/dods/GNUmakefile gdal-3.4.3-fedora-new/ogr/ogrsf_frmts/dods/GNUmakefile
--- gdal-3.4.3-fedora/ogr/ogrsf_frmts/dods/GNUmakefile	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/ogr/ogrsf_frmts/dods/GNUmakefile	2022-05-04 16:04:11.324217168 +0200
@@ -5,7 +5,7 @@ include ../../../GDALmake.opt
 OBJ	=	ogrdodsdriver.o ogrdodsdatasource.o ogrdodslayer.o \
 		ogrdodssequencelayer.o ogrdodsfielddefn.o ogrdodsgrid.o
 
-CPPFLAGS	:=	-iquote ..  $(CPPFLAGS) $(DODS_INC) $(LIBXML2_INC) -DDO_NOT_USE_DEBUG_BOOL
+CPPFLAGS	:=	-iquote ..  $(CPPFLAGS) $(DODS_INC) $(LIBXML2_INC) -DDO_NOT_USE_DEBUG_BOOL -I/usr/include/tirpc
 
 default:	$(O_OBJ:.o=.$(OBJ_EXT))
 
