diff -rupN --no-dereference gdal-3.4.3-fedora/doc/Makefile gdal-3.4.3-fedora-new/doc/Makefile
--- gdal-3.4.3-fedora/doc/Makefile	2022-04-22 11:02:04.000000000 +0200
+++ gdal-3.4.3-fedora-new/doc/Makefile	2022-05-04 16:04:11.767177773 +0200
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    ?= --keep-going -j auto -W
+SPHINXOPTS    ?= --keep-going
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
